kadi/commands/commands_v2.py

```diff
@@ -200,7 +200,7 @@ def get_cmds(
 
     :returns: CommandTable
     """
-    logger.info(f"Getting commands from {start} to {stop} for {scenario=}")
+    logger.info(f"Getting commands from {start.date} to {stop.date} for {scenario=}")
```

Comment:
```suggestion
    logger.info(f"Getting commands from {CxoTime(start).date} to {CxoTime(stop).date} for {scenario=}")
```

Reply:
Yep, that makes sense if start and stop can still be a string or a number.